Doctrine/RefreshTokenManager.php

```diff
@@ -41,12 +41,15 @@ class RefreshTokenManager implements RefreshTokenManagerInterface
     public function __construct(ObjectManager $om, $class)
     {
         $this->objectManager = $om;
-        $this->repository = $om->getRepository($class);
 
-        if (!$this->repository instanceof RefreshTokenRepositoryInterface) {
+        $repository = $om->getRepository($class);
+
+        if (!$repository instanceof RefreshTokenRepositoryInterface) {
```

Comment:
I made this tweak after doing a PHPStan analysis, this avoids setting the class property if the type is incorrect (if the property would typed, this would result in a type error instead of the exception below)